skll/learner.py

```diff
@@ -1071,9 +1072,19 @@ def model_params(self):
             # inverse transform to get indices for before feature selection
             coef = coef.reshape(1, -1)
             coef = self.feat_selector.inverse_transform(coef)[0]
-            for feat, idx in iteritems(self.feat_vectorizer.vocabulary_):
-                if coef[idx]:
-                    res[feat] = coef[idx]
+            if isinstance(self.feat_vectorizer, FeatureHasher):
+                self.logger.warning("No feature names are available since this model was trained on hashed features.")
+                num_features = len(coef)
+                index_width_in_feature_name = int(floor(log10(num_features))) + 1
```

Comment:
Won't `floor()` always return an integer anyway?

Reply:
That's what I had thought too but this is not the case for [Python 2](https://docs.python.org/2/library/math.html?highlight=math.floor#math.floor) which caused the python 2.7 tests to fail. Hence, the `int()` bit,.